src/help/guides/extension-development/function-definitions.md

```diff
@@ -1,13 +1,18 @@
 # Required Methods
 ## getMangaDetails
+### Method signature
 `async getMangaDetails(mangaId: string): Promise<Manga>`
 
-- A manga ID is defined by the source developer. However IDs are provided to Paperback in other functions, will be given to this function in the same format.
+### Parameters
+| Parameter | Type | Description|
+|-----------|------|------------|
+|  mangaId  | String | The ID of a manga. The manga ID is provided from the other discovery functions, such as the home page and performed searches. The manga ID should be used to return information about the manga, such as the title and author.|
 
-Given an ID, this async function should return a filled out Manga object.
+### Returns
+Given an ID, this async function should return a filled out [Manga](model-reference.md#Manga) object.
```

Comment:
We usually don't use file extension in links. Should we?
```suggestion
Given an ID, this async function should return a filled out [Manga](model-reference/#Manga) object.
```

Reply:
It doesn't make a difference from what I've seen.